bdd/spec/023_http_spec.sh

```diff
@@ -8,7 +8,7 @@ Describe "@std/http"
         from @std/http import get
 
         on start {
-          print(get('https://raw.githubusercontent.com/alantech/hellodep/aea1ce817a423d00107577a430a046993e4e6cad/index.ln'))
+          print(get('https://raw.githubusercontent.com/alantech/hellodep/aea1ce817a423d00107577a430a046993e4e6cad/index.ln').getOr('default'))
```

Comment:
This shouldn't be necessary, `print` can handle a `Result<Stringifiable>` just fine.

Reply:
I like that

Reply:
Transparently handling `Result` makes a lot of things simpler :) And when we Result-ify integer math, that property is going to persist :) going to make sure the built-in functions for dealing with integers can handle raw integers or Result-wrapped ones, so you can do some math and then print it and see if it worked or returned an error in the process no problem.